modules/server/src/main/scala/seqexec/server/tcs/GuideConfigDb.scala

```diff
@@ -186,6 +194,6 @@ object GuideConfigDb {
     )(TelescopeGuideConfig(_, _, _))(mountGuideDecoder, m1GuideDecoder, m2GuideDecoder)
 
   implicit val guideConfigDecoder: Decoder[GuideConfig] =
-    Decoder.forProduct2("tcsGuide", "gaosGuide")(GuideConfig)
+    Decoder.forProduct2("tcsGuide", "gaosGuide")(GuideConfig(_, _, false))
```

Comment:
Every time TCC sets a new guide configuration on GeMS, the pause/resume logic is reset. That is why the flag is set to `false`.